package.json

```diff
@@ -14,15 +14,12 @@
   "author": "Cory Tanner <cory.tanner@dockyard.com>, Aaron Sikes <aaron.sikes@dockyard.com> (https://dockyard.com)",
```

Comment:
Should we remove Aaron Sikes name or at least email?

Reply:
I'm not familiar with best practice around upkeep of author. Aaron helped develop the plugin in the very beginning but not the case past the initial creation of narwin-pack